test/images/README.md

```diff
@@ -182,7 +182,7 @@ After all the above has been done, run the [desired tests](https://github.com/ku
 
 Now that you have made the changes and tested locally, you are ready to share those changes.  This is a multi step process:
 
-1. Open a pull request with your changes to the test image (the new functionality and version bump). Go through the review process and merge the pull request. See this [pull request](https://github.com/kubernetes/kubernetes/pull/99860/files) for an example.
+1. Open a pull request to bump the version of your respective image. Each test image has a VERSION file in its directory. See [this pull request](https://github.com/kubernetes/kubernetes/pull/108771) for an example
```

Comment:
I think that this is ok, you submit the changes to the image and in the same PR you bump the version 
We forget to do it in yours

Reply:
(but I agree that the more usual scenario)

Reply:
I see what you mean, let me try to reword it

Reply:
yeah, I think changes and version bump must be tied, or we can overwrite images with same version and different functionality ... it was my fault, I should've verified that in your last PR

Reply:
I mean, if you build the image from current git history, it will create a 2.35 image with your changes ... and 2.35 already exist :)

Reply:
updated, PTAL